js/coinsbit.js

```diff
@@ -0,0 +1,1103 @@
+        //         id: '1595921',
+        //         time: '1640258718.402',
+        //         fee: '0.01572657',
+        //         price: '3931.64271609',
+        //         amount: '0.002',
+        //         role: '2',
+        //         deal: '7.86328543'
+        //      }
+        //
+        //  fetchClosedOrders
+        //      {
+        //         amount: '0.002',
+        //         price: '4000',
+        //         type: 'limit',
+        //         id: '8629354787',
+        //         side: 'buy',
+        //         ctime: '1640212275.928',
+        //         takerFee: '0.002',
+        //         ftime: '1640212275.93',
+        //         market: 'ETH_USDT',
+        //         makerFee: '0.002',
+        //         dealFee: '0.01597712',
+        //         dealStock: '0.002',
+        //         dealMoney: '7.98856357',
+        //         marketName: 'ETH_USDT'
+        //      },
+        const isFromFCOs = ('ctime' in order);
+        let id = undefined;
+        if (isFromFCOs) {
+            id = this.safeString (order, 'id');
+        } else {
+            id = this.safeString (order, 'orderId');
+        }
+        const side = this.safeString (order, 'side');
+        const orderType = this.safeString (order, 'type');
+        let timestamp = this.safeTimestamp2 (order, 'timestamp', 'time');
+        if (timestamp === undefined) {
+            timestamp = this.safeTimestamp2 (order, 'ctime', 'ftime');
+        }
+        const price = this.safeString (order, 'price');
+        const orderSize = this.safeString (order, 'amount');
+        const remaining = this.safeString (order, 'left');
+        const filledSize = this.safeString (order, 'dealStock');
+        const cost = this.safeString2 (order, 'dealMoney', 'deal');
+        const fee = this.safeString2 (order, 'dealFee', 'fee');
+        const marketId = this.safeString (order, 'market');
+        if (market === undefined) {
+            market = this.safeMarket (marketId, market, '_');
+        }
+        const symbol = this.safeString (market, 'symbol');
+        const quoteCurrency = market['quote'];
+        // <<<TODO>>> The below three lines are under question
+        // const feeCoeffTaker = this.safeString (order, 'takerFee');
+        const feeCoeffMaker = this.safeString (order, 'makerFee');
+        const willBeFee = Precise.stringMul (cost, feeCoeffMaker); // <<<TODO>> idk if the below 'willBeFee' & 'isFee' implementations make an useful attempt to calculate the actual fee (when using createOrder for limit order, which wasn't executed at market, and thus, has to be executed still). For example, when submiting limit-order (and it's not filled immediately as taker), then `dealFee` returns 0. however, it is still possible to calculate the 'dealFee' that will be when order will be triggered. (so, for limit orders its seems the only way to calculate the fee). however, I'll remove if this is not useful
```

Comment:
We should only reflect the factual truth, if the exchange says the fee is 0, we should not predict it, we should report 0. I would remove this, cause it will be dropped in `safeOrder` > `reduceFees`.